src/binding/petsc4py/src/petsc4py/PETSc/DMStag.pyx

```diff
@@ -38,22 +38,72 @@ class DMStagStencilLocation(object):
 # --------------------------------------------------------------------
 
 cdef class DMStag(DM):
-
+    """A DM object representing a "staggered grid" or a structured cell complex."""
     StencilType       = DMStagStencilType
     StencilLocation   = DMStagStencilLocation
 
-    def create(self, dim, dofs=None, sizes=None, boundary_types=None, stencil_type=None, stencil_width=None, proc_sizes=None, ownership_ranges=None, comm=None, setUp=False):
-        
+    def create(
+        self,
+        dim: int,
+        dofs: tuple[int, ...] | None = None,
+        sizes: tuple[int, ...] | None = None,
+        boundary_types: tuple[DM.BoundaryType, ...] | tuple[int, ...] | tuple[str, ...] | tuple[bool, ...] | None = None,
```

Comment:
```suggestion
        boundary_types: tuple[DM.BoundaryType | int | str | bool, ...] | None = None,
```

Reply:
Right. I can change it to be this way in other places.